designer/client/src/components/FieldEditors/TextFieldEdit.tsx

```diff
@@ -131,6 +131,8 @@ export function TextFieldEdit({ children, context = ComponentContext }: Props) {
         />
       </div>
 
+      {children}
```

Comment:
Moved this up so **Rows** is grouped with the other min/max options rather than after **Regex**